sonic_data_client/non_db_client.go

```diff
@@ -43,6 +43,10 @@ var (
 			path:    []string{"OTHERS", "platform", "cpu"},
 			getFunc: dataGetFunc(getCpuUtil),
 		},
+		{ // Get host uptime
+			path:    []string{"OTHERS", "proc", "sysuptime"},
```

Comment:
I think it is better to name this query `proc/uptime` (e.g. drop `sys`). The other queries in this category follow a particular convention where query path `proc/xyz` directly maps to file system `/proc/xyz`.

Reply:
Great suggestion! Updated.